op-node/sources/l1_client.go

```diff
@@ -151,13 +155,20 @@ func (s *L1Client) GoOrUpdatePreFetchReceipts(ctx context.Context, l1Start uint6
 						time.Sleep(3 * time.Second)
 						continue
 					}
-					_, _, err = s.FetchReceipts(ctx, blockInfo.Hash)
-					if err != nil {
-						s.log.Warn("failed to pre-fetch receipts", "err", err)
-						time.Sleep(200 * time.Millisecond)
+					waitErr := s.preFetchReceiptsRateLimiter.Wait(ctx)
```

Comment:
Is it a better way to put this ratelimit.wait before  L1BlockRefByNumber call? 
After triggering the rate limiting threshold, can reduce some unnecessary L1BlockRefByNumber calls

Reply:
The requests for L1BlockRefByNumber should all be necessary:
When we have not reached the latest block height, the request parameters for L1BlockRefByNumber are different every time, and the results obtained are useful.
When we reach the latest block height, continuously requesting L1BlockRefByNumber allows us to process subsequent processes as soon as a new block height appears. If we add a limiter here, we will still need to wait for a period of time before entering the subsequent process when a new block height appears.